integration/container/inspect_test.go

```diff
@@ -68,3 +70,34 @@ func TestInspectAnnotations(t *testing.T) {
 	assert.NilError(t, err)
 	assert.Check(t, is.DeepEqual(inspect.HostConfig.Annotations, annotations))
 }
+
+// TestNetworkAliasesAreEmpty verifies that network-scoped aliases are not set
+// for non-custom networks (network-scoped aliases are only supported for
+// custom networks, except for the "Default Switch" network on Windows).
+func TestNetworkAliasesAreEmpty(t *testing.T) {
+	ctx := setupTest(t)
+	apiClient := request.NewAPIClient(t)
+
+	netModes := []string{"host", "bridge", "none"}
+	if runtime.GOOS == "windows" {
+		netModes = []string{"nat", "none"}
+	}
+
+	for _, nwMode := range netModes {
+		t.Run(nwMode, func(t *testing.T) {
+			ctr := container.Run(ctx, t, apiClient,
```

Comment:
Silly question; must the container be running, or is "create" enough? (might save some time if we only have to create)

Reply:
If it must be running, you probably need something like this (but maybe there's code elsewhere for it because that code is in `integration-cli`); https://github.com/moby/moby/blob/178651733852a26bc11e6fa272b48ff37ecc7447/integration-cli/test_vars_test.go#L6-L11